xml/docu_styleguide.language.xml

```diff
@@ -88,6 +88,52 @@
    labels. This means that only the first word of any title should be capitalized.
    For example: <quote>Ceph core components</quote>.
   </para>
+  <para>
+    Use title-style capitalization for all book, article, and set titles.
+    This style is explained in <citetitle>The Chicago Manual of Style</citetitle>
+    8.167. For example: <quote>Deployment Guide</quote> or
+    <quote>Setting up a SAP HANA SR Cost Optimized Infrastructure</quote>.
```

Comment:
The latter example does not match the rules below, I think - shouldn't it be "Setting Up"? And how would English native speakers pronounce the abbreviation 'SAP'? German ones tend to pronounce it as "S-A-P" (three syllables), therefore I'm unsure if indefinite article should be "a" or "an"... maybe find another example?